packages/lodestar/src/db/syncCommitteeContribution.ts

```diff
@@ -143,11 +143,9 @@ function aggregateContributionInto(
     if (participated) {
       const syncCommitteeIndex = indexOffset + index;
       if (aggregate.syncCommitteeBits[syncCommitteeIndex] === true) {
-        throw new SyncContributionError({
-          code: SyncContributionErrorCode.ALREADY_KNOWN,
-          syncCommitteeIndex,
-          slot: contribution.slot,
-        });
+        // there are multiple SyncCommitteeContribution messaages of the same sub committee
+        // this means we aggregated a contribution of same sub committee already
+        return;
```

Comment:
I don't think we should return here. The consumer should be informed that we are not including this aggregate. The problem is that gossip handlers are not handled and they just reject unhandled.

I would add a try / catch here and return for that typed error like we do in the API
https://github.com/ChainSafe/lodestar/blob/3a63d7f8948234424335b8f4fae67e974259121d/packages/lodestar/src/network/gossip/handler.ts#L116

Reply:
if there are 10 contributions for the same subnet we'd ignore 9 of them and this is expected, why do you think we have to throw error here? I see that you refactor a lot in #2803 anyway so this is not a big deal.

actually we're not strictly following the spec `Of these contributions, proposers should select the best contribution seen across all aggregators for each subnet/subcommittee. A contribution with more valid signatures is better than a contribution with fewer signatures.`. Is this possible to do that in #2803 ?